src/TinyIoC/TinyIoC.cs

```diff
@@ -1650,6 +1650,30 @@ public bool Unregister(Type registerType, string name)
             return RemoveRegistration(typeRegistration);
         }
 
+        public void UnregisterMultiple(Type registrationType, IEnumerable<Type> implementationTypes)
+        {
+            if (implementationTypes == null)
+                throw new ArgumentNullException("implementationTypes", "types is null.");
+
+            foreach (var type in implementationTypes)
+//#if NETFX_CORE
+//              if (!registrationType.GetTypeInfo().IsAssignableFrom(type.GetTypeInfo()))
+//#else
+                if (!registrationType.IsAssignableFrom(type))
+//#endif
+                    throw new ArgumentException(String.Format("types: The type {0} is not assignable from {1}", registrationType.FullName, type.FullName));
+
+            foreach (var type in implementationTypes)
+            {
+                Unregister(registrationType, type.FullName);

```

Comment:
This is the key line of code. It implements the same `type.FullName` convention that `RegisterMultiple` does.